tests/ssl-opt.sh

```diff
@@ -55,6 +56,9 @@ get_options() {
             -m|--memcheck)
                 MEMCHECK=1
                 ;;
+            --seed)
+                shift; SEED="$1"
```

Comment:
How is that different from / preferable over `FILTER=$1` a few lines above?

Reply:
I don't understand the question. Both options take arguments, and they put their arguments in different variables.

Reply:
Sorry, it was just a bash question - there are quotes for `seed` and none for `filter`.

Reply:
Oh ok. Here the quotes don't make any difference. The rules for when quotes are necessary are a bit subtle, so I prefer to recommend using them even where they're optional, but it's a bit like systematically using braces on `if () …` in C — it isn't a universally recognized style rule.